generate-data/src/main.rs

```diff
@@ -79,22 +79,15 @@ fn build_electron_to_chromium() -> Result<()> {
         (electron_version.parse::<f32>().unwrap(), chromium_version)
     })
     .collect::<Vec<_>>();
-    data.sort_by(|(a, _), (b, _)| a.partial_cmp(b).unwrap());
-    let data = data
-        .into_iter()
-        .map(|(electron_version, chromium_version)| {
-            quote! {
-                (#electron_version, #chromium_version)
-            }
-        });
+    data.sort_by(|(a, _), (b, _)| a.total_cmp(b));
+    let (electron_versions, chromium_versions): (Vec<_>, Vec<_>) = data.into_iter().unzip();
 
-    fs::write(
-        path,
-        quote! {{
-            vec![#(#data),*]
-        }}
-        .to_string(),
-    )?;
+    let code = quote! {
+        static ELECTRON_VERSIONS: &'static [f32] = &[ #(#electron_versions),* ];
```

Comment:
We split the unaligned types into different arrays to reduce size, because `(f32, &str)` will waste 32bits of space per entry.